cli/tests/integration/coverage_tests.rs

```diff
@@ -51,76 +54,63 @@ fn error_if_invalid_cache() {
   std::fs::copy(mod_test_path, mod_test_temp_path).unwrap();
 
   // Generate coverage
-  let status = util::deno_cmd_with_deno_dir(&deno_dir)
-    .current_dir(deno_dir_path)
-    .arg("test")
-    .arg("--quiet")
-    .arg(format!("--coverage={}", tempdir.to_str().unwrap()))
-    .stdout(std::process::Stdio::piped())
-    .stderr(std::process::Stdio::inherit())
-    .status()
-    .unwrap();
-
-  assert!(status.success());
+  let output = context
+    .new_command()
+    .args(format!(
+      "test --quiet --coverage={}",
+      tempdir.to_str().unwrap()
+    ))
+    .run();
+
+  output.assert_exit_code(0);
+  output.skip_output_check();
 
   // Modify the file between deno test and deno coverage, thus invalidating the cache
   std::fs::copy(mod_after_path, mod_temp_path).unwrap();
 
-  let output = util::deno_cmd_with_deno_dir(&deno_dir)
-    .current_dir(deno_dir_path)
-    .arg("coverage")
-    .arg(format!("{}/", tempdir.to_str().unwrap()))
-    .stdout(std::process::Stdio::piped())
-    .stderr(std::process::Stdio::piped())
-    .output()
-    .unwrap();
+  let output = context
+    .new_command()
+    .args(format!("coverage {}/", tempdir.to_str().unwrap()))
+    .run();
 
-  assert!(output.stdout.is_empty());
+  output.assert_exit_code(1);
+  let out = output.combined_output();
 
   // Expect error
-  let error =
-    util::strip_ansi_codes(std::str::from_utf8(&output.stderr).unwrap())
-      .to_string();
+  let error = util::strip_ansi_codes(out).to_string();
```

Comment:
Maybe we could add a `pub fn no_color(&mut self) -> &mut Self { self.env("NO_COLOR", "1") }` method to `TestCommandBuilder` and use that above? Then we could get rid of all these strip_ansi_codes calls.

That said, it seems tempting to just have it always on be default and make it opt out...